config/nuxt/build.js

```diff
@@ -12,10 +12,11 @@ module.exports = {
   ** Run ESLint on save
   */
   extend (config, { isDev, isClient }) {
-    // remove SVG from URL loader, so vue-svg-loader can be used for SVGs instead
-    // based on https://github.com/nuxt/nuxt.js/issues/1332#issuecomment-321694185
-    const urlLoader = config.module.rules.find((rule) => rule.loader === 'url-loader')
-    urlLoader.test = /\.(png|jpe?g|gif)$/
+    config.module.rules.forEach((rule) => {
+      if (rule.test.toString() === '/\\.(png|jpe?g|gif|svg|webp)$/') {
+        rule.test = /\.(png|jpe?g|gif|webp)$/
+      }
+    })
 
     if (isDev && isClient) {
```

Comment:
`isClient` does not exist in Nuxt 2 anymore, use `process.client` (https://medium.com/nuxt/nuxt-2-is-coming-oh-yeah-212c1a9e1a67#b740)